### DISCUSSION TITLE
Welcome to gameOS Discussions!

### DISCUSSION THREAD

**Discussion by PlayingKarrde** [PlayingKarrde/gameOS#160] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alfredolvera** [comment]
Hey @PlayingKarrde!

Thanks for all your work with Pegasus. The themes you've created are nothing short of spectacular!

I guess I would like to ask some questions about GameOS, about things I've been trying to implement in my fork without success:

- 90 degrees-cornered tiles are a bit out of date in the UI design world (JK xD), so I would love to add some rounded corners to your theme. I've been trying to do it, but I'm struggling with the video frame and the fade transition between the background and the video itself for the tiles. If you have any ideas or code that I could test, it would be very helpful.

- I have also been trying to add a tile called "All Games" in the platform horizontal list that does just that: a list of all your installed games on one page, regardless of platform. I'm just new to QML and I can't get it to work. 

- The other thing I've been trying to implement in my fork is user customization. I already implemented a very crude but functional color selector for highlights and background, but I would love to implement a user-selectable image or video to use as a background, so we have all the customization that a console-like theme would have, but I'm also clueless to how to implement it. I've tried but failed so far with my code.

Can't wait to see what you come up with in the future with updates to your themes. Keep rocking the Pegasus world, dude!

**Comment by code-h0** [comment]
Hi @PlayingKarrde 

I think an introvideo or bootvideo would be cool. For example: by placing a videofile (boot.mp4) into the gameOS folder and play it after starting the frontend an cancel it if a button is pressed.

**Reply by wickedTangent** [reply]
This would also help during long boot time (due to large amount of metadata and media

**Reply by PlayingKarrde** [reply]
Hmm it wouldn't be hard to integrate but I don't have a lot of time for making one right now. I'll have a think about it though.

**Comment by TwoShedsWilson** [comment]
This is exactly what I've been looking for! Please keep up the good work!
